### PR TITLE
Accelerometer with Gyroscope publisher

### PR DESCRIPTION
**Pull request by krishnachaitanya7**
- Add Gyro data to accelerometer publisher

### CODE REVIEW

File: scripts/publishers/accelerometer_publisher.py [hunk 1]
```diff
@@ -12,16 +13,20 @@
     args = parser.parse_args()
     ros_robotic_skin_path = rospkg.RosPack().get_path('ros_robotic_skin')
     config_file = ros_robotic_skin_path + "/config/" + args.config_file
-    accel = LSM6DS3_acclerometer(config_file)
-    rospy.init_node('talker_%s' % str(accel.config_dict['imu_number']), anonymous=True)
-    pub = rospy.Publisher('/imu_data%s' % str(accel.config_dict['imu_number']), Imu, queue_size=10)
-    r = rospy.Rate(100)
```
Comment (watakandai) [inline scripts/publishers/accelerometer_publisher.py:18]: why did you delete this line and use `sleep(0.1)` instead?

**Review by watakandai — CHANGES_REQUESTED** [review]

File: scripts/publishers/accelerometer_publisher.py (hunk 1 above)
Comment (krishnachaitanya7) [reply scripts/publishers/accelerometer_publisher.py:18]: So rospy.sleep was apparently giving problems, sometimes it gives problems and sometimes it doesn't. So I thought to replace it with python sleep as it always works without fail

**Review by peasant98 — APPROVED** [review]
lgtm

File: scripts/publishers/accelerometer_publisher.py [hunk 2]
```diff
@@ -12,16 +13,20 @@
...
+    accel_gyro = LSM6DS3_IMU(config_file)
+    rospy.init_node('talker_%s' % str(accel_gyro.config_dict['imu_number']), anonymous=True)
```
Comment (peasant98) [inline scripts/publishers/accelerometer_publisher.py]: consider `.format` here.

**Review by bjkim00 — APPROVED** [review]

File: scripts/publishers/accelerometer_publisher.py (hunk 2 above)
Comment (krishnachaitanya7) [reply scripts/publishers/accelerometer_publisher.py]: got it. will do.

Comment (krishnachaitanya7) [reply scripts/publishers/accelerometer_publisher.py]: Done!

**Review by watakandai — APPROVED** [review]

**Comment by krishnachaitanya7** [comment]
Merging...

File: scripts/publishers/accelerometer_publisher.py (hunk 1 above)
Comment (watakandai) [reply scripts/publishers/accelerometer_publisher.py:18]: oh I think it's because you haven't called `rospy.init` before `LSM6DS3_IMU`

**Pull request merged**